src/backend.c

```diff
@@ -137,26 +137,31 @@ static int ocl_check_dri (MAYBE_UNUSED hashcat_ctx_t *hashcat_ctx)
 
   // Now we need to check if this an AMD vendor, because this is when the problems start
 
-  FILE *fd_drm = fopen (drm_card0_vendor_path, "rb");
+  HCFILE fp_drm;
+//  FILE *fd_drm = fopen (drm_card0_vendor_path, "rb");
```

Comment:
Please use the format which you've commented out

Reply:
I need to switch back to fopen in "ocl_check_dri" ? There are similar functions for nvidia too, do I have to change those too?